api/backups/remove.go

```diff
@@ -4,15 +4,12 @@
 package backups
 
 import (
-	"github.com/juju/errors"
-
 	"github.com/juju/juju/apiserver/params"
 )
 
-func (c *Client) Remove(id string) error {
-	args := params.BackupsRemoveArgs{ID: id}
-	if err := c.facade.FacadeCall("Remove", args, nil); err != nil {
-		return errors.Trace(err)
-	}
-	return nil
+func (c *Client) Remove(ids []string) []params.ErrorResult {
+	args := params.BackupsRemoveArgs{IDs: ids}
+	var results params.ErrorResults
+	c.facade.FacadeCall("Remove", args, &results)
```

Comment:
I think this maybe wrong? I think you would still want to check the return error from the `FacadeCall`. For example the `FacadeCall` can return `too many retries` from the `APICall` function and you'd probably want to know about that?

Reply:
done